dashboard_description_plugin/scripts/refresh-md.js

```diff
@@ -1,15 +1,20 @@
 #!/usr/bin/env node
 // (C) 2022-2023 GoodData Corporation
-/* eslint-disable @typescript-eslint/no-var-requires */
 
-const dotenv = require("dotenv");
-const path = require("path");
+import path from "path";
+import { fileURLToPath } from "url";
+
+import dotenv from "dotenv";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
 dotenv.config({ path: path.resolve(__dirname, "../.env") });
 dotenv.config({ path: path.resolve(__dirname, "../.env.secrets") });
 
 const workspace = process.env.WORKSPACE;
 const hostname = process.env.BACKEND_URL;
-const backend = process.env.BACKEND_TYPE || 'tiger';
+const backend = process.env.BACKEND_TYPE || "bear";
```

Comment:
@dremileta we want to keep the default tiger

Reply:
@dremileta when updating the internal repo (as asked here https://github.com/gooddata/gooddata-dashboard-plugins/pull/48#discussion_r1415523296), please bring this change there as well for consitency.

Reply:
@PavelJiranek done...and yeah, I already brought the changes to internal first